### PR TITLE
Added LB TVL for Magicsea DEX

### PR DESCRIPTION
**Pull request by 0xjosaphat**

### CODE REVIEW

**Comment by llamatester** [comment]
The adapter at projects/magicsea-lb-v2 exports TVL: 
        
 
 
    shimmer_evm               220.08 k
    iotaevm                   45.17 k
    
    total                    265.25 k

File: projects/magicsea/index.js
Comment (g1nt0ki) [inline projects/magicsea/index.js:1]: can you revert this change, folder names are hardcoded and will break our server

**Comment by llamatester** [comment]
The adapter at projects/magicsea-lb-v2 exports TVL: 
        
 
 
    shimmer_evm               250.21 k
    iotaevm                   40.65 k
    
    total                    290.86 k

**Comment by llamatester** [comment]
The adapter at projects/magicsea-lb-v2 exports TVL: 
        
 
 
    shimmer_evm               250.31 k
    iotaevm                   40.52 k
    
    total                    290.83 k

File: projects/magicsea/index.js
Comment (0xjosaphat) [reply projects/magicsea/index.js:1]: i moved it back to /shimmersea, cheers

File: projects/magicsea-lb-v2/index.js [hunk 1]
```diff
@@ -0,0 +1,37 @@
+const {sumTokens2} = require('../helper/unwrapLPs')
+
+const factories = {
+    shimmer_evm: '0xEE0616a2DEAa5331e2047Bc61E0b588195A49cEa',
```
Comment (g1nt0ki) [inline projects/magicsea-lb-v2/index.js:4]: noticed that we already count this as part of swapline, so excluded it

Comment (0xjosaphat) [reply projects/magicsea-lb-v2/index.js:4]: Hi @g1nt0ki MagicSea and Swapline are from the same team. After your comment we had a discussion in our team and we agreed to have the LB metrics for Shimmer on Swapline removed in favour to show them in MagicSea. 
Could you please take care of this? Or do you prefer that I create one PR in Swapline and one PR in MagicSea?

Comment (0xjosaphat) [reply projects/magicsea-lb-v2/index.js:4]: Another question is that MagicSea and MagicSea LB are currently treated as two separate projects. Unlike e.g. TraderJoe where the LB part is a subproject of TraderJoe. 
Can you give us advice what we need to do to have MagicSea LB visible in a tab within the "main" MagicSea DEX?

Comment (g1nt0ki) [reply projects/magicsea-lb-v2/index.js:4]: https://defillama.com/protocol/magicsea here is the parent protocol with the both the sub protocols :)
